### PR TITLE
vendor: update containerd to 08f7ee98

### PR DESCRIPTION
**Pull request by tonistiigi**
bring in the arm platform matching fixes

https://github.com/containerd/containerd/pull/2414
https://github.com/containerd/containerd/pull/2417

Signed-off-by: Tonis Tiigi <tonistiigi@gmail.com>

### CODE REVIEW

**Comment by andrewhsu** [comment]
comparison of upstream changes across the vendor git commit range: https://github.com/containerd/containerd/compare/63522d9...08f7ee9

@tiborvass when you get a chance, can you double-check this PR?

**Review by thaJeztah — APPROVED** [review]
LGTM

File: vendor.conf [hunk 1]
```diff
@@ -114,7 +114,7 @@ github.com/googleapis/gax-go v2.0.0
 google.golang.org/genproto 694d95ba50e67b2e363f3483057db5d4910c18f9
 
 # containerd
-github.com/containerd/containerd 63522d9eaa5a0443d225642c4b6f4f5fdedf932b
+github.com/containerd/containerd 08f7ee9828af1783dc98cc5cc1739e915697c667
```
Comment (tiborvass) [inline vendor.conf]: @tonistiigi shouldn't we sync `hack/dockerfile/install/containerd.installer` with this as well?

Comment (tonistiigi) [reply vendor.conf]: We only use release versions in there. @dmcgowan will update that to `v1.1.1-rc.2` when it is released.

Comment (tiborvass) [reply vendor.conf]: Alright

**Review by tiborvass — APPROVED** [review]
LGTM

**Pull request merged**